x-pack/plugins/observability/server/services/slo/summary_transform/helpers/create_temp_summary.ts

```diff
@@ -25,7 +26,8 @@ export function createTempSummaryDocument(slo: SLO) {
         duration: slo.timeWindow.duration.format(),
         type: slo.timeWindow.type,
       },
-      instanceId: '*',
+      groupBy: !!slo.groupBy ? slo.groupBy : ALL_VALUE,
+      instanceId: ALL_VALUE,
```

Comment:
Since we don't know the values behind the groupBy, we default to '*' value, but it will be deleted when the real summary documents are indexed